{"name": "Verify Get Request Status Code", "status": "failed", "statusDetails": {"message": "AssertionError: assert 404 == 400\n +  where 404 = <Response [404]>.status_code\n +  and   400 = int('400')", "trace": "self = <data_driven_test.test_data_driven.Test_DDT_URL object at 0x000001CF4DA0D420>, url = 'http://supplant.com/bot', expected_status_code = '400'\n\n    @allure.title(\"Verify Get Request Status Code\")\n    @allure.description(\"This test verify that the request passed successfully\")\n    @pytest.mark.parametrize(keys, values)\n    def test_status_code(self, url, expected_status_code):\n        response = requests.get(url)\n>       assert response.status_code == int(expected_status_code)\nE       AssertionError: assert 404 == 400\nE        +  where 404 = <Response [404]>.status_code\nE        +  and   400 = int('400')\n\ndata_driven_test\\test_data_driven.py:17: AssertionError"}, "description": "This test verify that the request passed successfully", "parameters": [{"name": "url", "value": "'http://supplant.com/bot'"}, {"name": "expected_status_code", "value": "'400'"}], "start": 1651386964476, "stop": 1651386971054, "uuid": "84bd22c9-6a20-438b-aff8-13162e13dbb5", "historyId": "fed501b087345b11fd6ed531ca2ce706", "testCaseId": "55499a46d297655c67691cd3888cc7f1", "fullName": "data_driven_test.test_data_driven.Test_DDT_URL#test_status_code", "labels": [{"name": "parentSuite", "value": "data_driven_test"}, {"name": "suite", "value": "test_data_driven"}, {"name": "subSuite", "value": "Test_DDT_URL"}, {"name": "host", "value": "AlexK-Dell"}, {"name": "thread", "value": "3712-MainThread"}, {"name": "framework", "value": "pytest"}, {"name": "language", "value": "cpython3"}, {"name": "package", "value": "data_driven_test.test_data_driven"}]}